IMAGES.rst

```diff
@@ -39,19 +39,19 @@ The images are named as follows:
 
 where:
 
-* ``BRANCH_OR_TAG`` - branch or tag used when creating the image. Examples: ``master``, ``v1-10-test``, ``1.10.14``
-  The ``master`` and ``v1-10-test`` labels are built from branches so they change over time. The ``1.10.*`` and in
-  the future ``2.*`` labels are build from git tags and they are "fixed" once built.
-* ``PYTHON_MAJOR_MINOR_VERSION`` - version of python used to build the image. Examples: ``3.5``, ``3.7``
+* ``BRANCH_OR_TAG`` - branch or tag used when creating the image. Examples: ``master``,
+  ``v2-0-test``, ``v1-10-test``, ``2.0.0``. The ``master``, ``v1-10-test`` ``v2-0-test`` labels are
+  built from branches so they change over time. The ``1.10.*`` and ``2.*`` labels are built from git tags
+  and they are "fixed" once built.
+* ``PYTHON_MAJOR_MINOR_VERSION`` - version of python used to build the image. Examples: ``3.6``, ``3.7``
 * The ``-ci`` suffix is added for CI images
-* The ``-manifest`` is added for manifest images (see below for explanation of manifest images)
 
 We also store (to increase speed of local build/pulls) python images that were used to build
 the CI images. Each CI image, when built uses current python version of the base images. Those
 python images are regularly updated (with bugfixes/security fixes), so for example python3.8 from
 last week might be a different image than python3.8 today. Therefore whenever we push CI image
 to airflow repository, we also push the python image that was used to build it this image is stored
-as ``apache/airflow:python-3.8-<BRANCH_OR_TAG>``.
+as ``apache/airflow:python<PYTHON_MAJOR_MINOR_VERSION>-<BRANCH_OR_TAG>``.
```

Comment:
Just double-check: it was written as `python-3.8-<BRANCH_OR_TAG>` but now as `python<PYTHON_MAJOR_MINOR_VERSION>-<BRANCH_OR_TAG>`, i.e., a dash is removed. I assume this is a fix? (because among the tags on DockerHub and the samples below, I don't see dash between `python` and python version number)

Reply:
Yep. It is a fix :)